superset-frontend/src/dashboard/components/SliceHeaderControls/index.jsx

```diff
@@ -302,7 +302,11 @@ class SliceHeaderControls extends React.PureComponent {
             triggerNode.closest(SCREENSHOT_NODE_SELECTOR)
           }
         >
-          <span id={`slice_${slice.slice_id}-controls`} role="button">
+          <span
+            id={`slice_${slice.slice_id}-controls`}
+            role="button"
+            aria-label="..."
```

Comment:
Should this be something more accessible (for screen readers and such) like "More Options"?

Reply:
Done!